mobile/lib/shared/services/api.service.dart

```diff
@@ -70,6 +76,29 @@ class ApiService {
     return url;
   }
 
+  Future<bool> _isEndpointAvailable(String serverUrl) async {
```

Comment:
I believe we would hit the .well-known endpoint to get the `/api` if the user forgot to put it in. Maybe only applicable to https but not local IP, can you help me double check

Reply:
Lmk if you have any more question after this after the discussion on Discord